docs/source/guides/user/chapters/datadirs.rst

```diff
@@ -0,0 +1,55 @@
+Avocado Data Directories
+========================
+
+When running tests, we are frequently looking to:
+
+* Locate tests
+* Write logs to a given location
+* Grab files that will be useful for tests, such as ISO files or VM disk
+  images
+
+Avocado has a module dedicated to find those paths, to avoid cumbersome
```

Comment:
I think this reads better: "dedicated to finding those paths"

Reply:
This was already there, I just moved places, but sure. I will fix it.